### PR TITLE
Document the apos.images.srcset template helper

### PR DESCRIPTION
**Pull request by fredrikekelund**
As mentioned in https://github.com/punkave/apostrophe/pull/805, here's some documentation for apostrophe-images-widgets' `sizesAttr` option. I added it to the "Adding editable content to your pages" chapter. I don't know if the tonality hit home, feel free to edit it as you please otherwise!

### CODE REVIEW

**Comment by fredrikekelund** [comment]
This PR was opened as a complement to apostrophecms/apostrophe#805. But since the outcome of that PR was not to add a srcset feature directly to apostrophe-images-widgets (but rather to introduce just a template helper `apos.images.srcset`), I've changed this PR to simply add a section to the "Responsive images" how-to about how to write a simple widget that renders an `` tag with a srcset attribute.

**Comment by boutell** [comment]
Thanks, this is great.

File: tutorials/howtos/responsive-images.md [hunk 1]
```diff
@@ -3,7 +3,62 @@ title: "Responsive images"
 layout: tutorial
 ---
 
-The `apostrophe-images` widget provides a powerful way to select and display images on the site, including the ability to manually crop images. But sometimes, this isn't best for mobile-responsive site designs.
+The `apostrophe-images-widgets` module provides a powerful way to select and display images on the site, including the ability to manually crop images on the server or responsively on the client.
```
Comment (boutell) [inline tutorials/howtos/responsive-images.md]: I don't think the language about "manually on the server or responsively on the client" is clear. How about:

"Including the ability to manually crop images, or rely on CSS to carry out various automatic responsive cropping approaches."

File: tutorials/howtos/responsive-images.md [hunk 2]
```diff
@@ -3,7 +3,62 @@ title: "Responsive images"
...
+
+However, this dynamic behavior comes at the expense of not rendering images as `<img>` tags, but rather as CSS based background images. If you're looking to use the `srcset` attribute, you can do so by loading a custom widget.
+
+## An image widget that supports `srcset`
+
+The `srcset` attribute was introduced to give developers a way to declaratively specify which sizes of an image are available on the server. It is coupled with the `sizes` attribute that lets us specify how much space the image if supposed to take up in the browser. Eric Portis has an excellent [explanation](https://ericportis.com/posts/2014/srcset-sizes/) on his blog.
+
+Apostrophe lets use make use of this browser feature through the `apos.images.srcset` template helper. In the following example, we load a simple custom widget that renders and image with both a `srcset` and a `sizes` attribute.
+
+> Remember that it's up to the developer to specify the `sizes` attribute, so when you load the widget into an area or singleton, follow this example:
```
Comment (boutell) [inline tutorials/howtos/responsive-images.md]: Should be "sizesAttr" presumably.

**Comment by boutell** [comment]
@fredrikekelund I know I merged but I spotted a couple things after; mind checking those out?

**Comment by fredrikekelund** [comment]
Sure, will have a look and open a new PR as soon as I get a chance!

**Pull request merged**